### DISCUSSION TITLE
Proposal: Night Mode / Time dependant brightness

### DISCUSSION THREAD

**Discussion by qbinary** [InfiniTimeOrg/InfiniTime#1606] [Ideas]
I would really like to see something like a night mode or a time dependant brightness value so that the PineTime dims down at night and goas back to full (/last set) brightness in the morning.

**Comment by JF002** [comment]
The pinetime does not have a luminosity sensor, but increasing / decreasing the backlight at some pre-defined (or configurable) time should not be difficult to implement.

**Comment by qbinary** [comment]
I really would like to implement this feature but I am currently very bussy with my abitur exams and do not have enought time to get warm with infinitime. I already have my devkit hooked up and successfully compiled and flashed infinitime but until my last exam I have to restrict my contributions to issues :(

**Comment by KoalaV2** [comment]
Found this issue relevant so including my suggestion here. Making the display yellow / darker towards the night would also be nice since all my other devices have this and opening a bright blue / white display during the night to check the time is a bit annoying.

**Comment by Eloitor** [comment]
I'm also interested in this feature.

**Comment by Riksu9000** [comment]
For me the brightness setting is very situational and hardly time dependant. How would overriding the brightness be handled? Wouldn't it be annoying if it just changed on it's own after you specifically set it to some level?
